.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  integ-tests:
+    runs-on: ubuntu-latest
+    environment:
+      name: prod
+    steps:
+        - uses: actions/checkout@v3
```

Comment:
I imagine most of the examples out there use `@v3`, but `@v4` is the [latest version available](https://github.com/actions/checkout), and it avoids this warning that you can see in [your runs](https://github.com/mbfreder/aws-lambda-runtime-interface-emulator/actions/runs/7995969160):

```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

Reply:
Updated.